seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobMaster.java

```diff
@@ -274,6 +275,11 @@ private CheckpointConfig createJobCheckpointConfig(
             jobCheckpointConfig.setCheckpointInterval(
                     Long.parseLong(
                             jobEnv.get(EnvCommonOptions.CHECKPOINT_INTERVAL.key()).toString()));
+        } else if (jobEnv.containsKey(EnvCommonOptions.JOB_MODE.key())
```

Comment:
`JOB_MODE `

What is the default value?

Reply:
> `JOB_MODE `
> 
> What is the default value?

if not set in env, it should be empty in here.

Reply:
this default value is  batch